apps/web/src/app/dashboard/[name]/settings/general/page.tsx

```diff
@@ -65,29 +71,32 @@ function SettingsGeneral({ params }: SettingsMembersViewProps) {
     },
   });
 
-
-
-  const { data: userData, refetch: refetchUsers, isFetched: usersLoaded } = useQuery({
+  const {
+    data: userData,
+    refetch: refetchUsers,
+    isFetched: usersLoaded,
+  } = useQuery({
     queryKey: ["members", organization?.id],
     queryFn: () => getMembersOfOrganization(organization!.id),
     enabled: !!organization?.id && isFetched,
```

Comment:
If we want to check if it was fetched we should check it first. Because if there is an org or it equals to `null` `isFetched` will always evaluate to `true`. 
```suggestion
    enabled: isFetched && !!organization?.id,
```
Or you can delete this check entirely.